Tests/BuildTests/ClangTargetBuildDescriptionTests.swift

```diff
@@ -49,8 +49,33 @@ final class ClangTargetBuildDescriptionTests: XCTestCase {
 
     private func makeTargetBuildDescription() throws -> ClangTargetBuildDescription {
         let observability = ObservabilitySystem.makeForTesting(verbose: false)
+
+        let manifest = Manifest.createRootManifest(
+            displayName: "dummy",
+            toolsVersion: .v5,
+            targets: [try TargetDescription(name: "dummy")]
+        )
+
+        let target = try makeResolvedTarget()
+
+        let package = Package(identity: .plain("dummy"),
+                              manifest: manifest,
+                              path: .root,
+                              targets: [target.underlying],
+                              products: [],
+                              targetSearchPath: .root,
+                              testTargetSearchPath: .root)
+
         return try ClangTargetBuildDescription(
-            target: try makeResolvedTarget(),
+            package: .init(underlying: package,
+                           defaultLocalization: nil,
+                           supportedPlatforms: [],
+                           dependencies: [],
+                           targets: [target],
+                           products: [],
+                           registryMetadata: nil,
+                           platformVersionProvider: .init(implementation: .minimumDeploymentTargetDefault)),
+            target: target,
```

Comment:
Can a new test or something on top of existing tests be added to verify that `-I` is actually passed when targeting Linux or Android?

Reply:
It might be tricky because I don’t know if we have `toolchain` setup in tests, but I will give it a shot!

Reply:
I don't think we need a full toolchain for build plan tests, passing build parameters is usually sufficient, and there's a `mockBuildParameters` helper function for that.

Reply:
Fixed!